src/main/kotlin/graph/AdjacencyList.kt

```diff
@@ -10,67 +10,67 @@ abstract class AdjacencyList(initiallyVertexCount: Int = 0) {
     protected val adjacencyList = ArrayList<ArrayList<Edge>>()
```

Comment:
it causes an error if we try to call adjacencyList() in UndirectedGraph
put this string before init {}

Reply:
done